task/06-conditions-n-loops-tasks.js

```diff
@@ -82,7 +103,14 @@ function getSumBetweenNumbers(n1, n2) {
  *   10,10,10 =>  true
  */
 function isTriangle(a,b,c) {
-    throw new Error('Not implemented');
+    if(a + b <= c) {
+        return false;
+    } else if (b + c <= a) {
+        return false;
+    } else if (c + a <= b) {
+        return false;
+    }
+    return true;
```

Comment:
We can just `return (a + b >= c) && (a + c >= b) && (c + b >= a)`
Please, see https://javascript.info/logical-operators
Please, read about https://developer.mozilla.org/en-US/docs/Glossary/Truthy